src/tools/idlc/tests/type_meta.c

```diff
@@ -18,11 +18,11 @@
 #include "dds/ddsi/ddsi_typelib.h"
 #include "dds/ddsi/ddsi_typewrap.h"
 #include "dds/ddsc/dds_opcodes.h"
+#include "libidlc/descriptor_type_meta.h"
 #include "descriptor_type_meta.h"
```

Comment:
How about prefixing header files similar to DDSC and DDSI (i.e. `idl_descriptor.h`, but also for source files) so that we can avoid duplicate file names? (private header files will be prefixed with `idl__`)

Reply:
Did this in the newly pushed version